docs/revsets.md

```diff
@@ -12,9 +12,10 @@ commits) to such commands.
 
 The words "revisions" and "commits" are used interchangeably in this document.
 
-The commits listed by `jj log` without arguments are called "visible commits".
-Other commits are only included if you explicitly mention them (e.g. by commit
-ID or a Git ref pointing to them).
+Most revsets search only the [visible commits](glossary.md#visible-commits).
+Other commits (for example, abandoned or [rewritten](glossary.md#rewrite)
```

Comment:
We can delete the parenthetical if we are OK with the following commit (putting the same info in glossary.md).